README.md

```diff
@@ -23,6 +23,36 @@ model behaviors and take the mystery out of model development. Kolena helps you:
 This `kolena` package contains the Python client library for programmatic interaction with the Kolena ML testing
 platform.
 
+## Setup
+
+Client builds can be installed directly from PyPI using any Python package manager such as pip:
+
+```zsh
+pip install kolena
+```
+
+Advanced use cases (eg. metrics computation) may require extra dependencies which can be installed by running:
+```zsh
+pip install `kolena[metrics]`
```

Comment:
Remove backticks — quotes around this shouldn't be necessary in common shells, and backticks are separate thing entirely.